fastcrypto-vdf/src/math/crt.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::math::extended_gcd::extended_euclidean_algorithm;
+use num_bigint::BigInt;
+use num_integer::Integer;
+use num_traits::{One, Signed};
+
+/// Find the unique x such that x = a mod p and x = b mod q for relatively prime p and q and 0 <= x
+/// < pq.
+pub fn solve_simple_equation(a: &BigInt, p: &BigInt, b: &BigInt, q: &BigInt) -> Option<BigInt> {
```

Comment:
solve_crt maybe?

Reply:
Renamed to `solve_congruence_equation_system`. A bit long but more precise.